hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoop/mapreduce/InputFormatBuilder.java

```diff
@@ -124,6 +108,15 @@
    * @since 2.0
    */
   interface InputFormatOptions<T> {
+    /**
+     * Sets the {@link Authorizations} used to scan. Must be a subset of the user's authorizations.
+     * By Default, auths are set to {@link Authorizations#EMPTY}
```

Comment:
This doesn't match the new client API, which defaults to the maximal set for the user.

Reply:
Where is this done?

Reply:
https://github.com/apache/accumulo/blob/ed505d076071c792ca97441d17ae2ebd00c34a17/core/src/main/java/org/apache/accumulo/core/clientImpl/AccumuloClientImpl.java#L129-L134
and
https://github.com/apache/accumulo/blob/ed505d076071c792ca97441d17ae2ebd00c34a17/core/src/main/java/org/apache/accumulo/core/clientImpl/AccumuloClientImpl.java#L204-L209